### PR TITLE
build: add makefile and start scripts

### PR DESCRIPTION
**Pull request by Krzysztofoc**
### Change description

Adding Makefile, start.sh, version and routes.yml to make pipeline working properly with already existing pipeline jobs definitions.

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are commited to keeping commit history clean, consistent and linear. To achive this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: version [hunk 1]
```diff
@@ -0,0 +1 @@
+0.1
```
Comment (geek-soft) [inline version]: How is that file used?

File: routes.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+app_name: lfp-appeals-frontend
+group: web
+weight: 900
+routes:
+  1: /appeal-a-penalty
```
Comment (geek-soft) [inline routes.yaml]: Where did you take it from?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+.PHONY: clean
+clean:
+	rm -f ./$(artifact_name)-*.zip
+	rm -rf ./build
+	rm -rf ./temp-*
+	rm -f ./build.log
+
+package-install:
+	npm i
```
Comment (geek-soft) [inline Makefile]: I suggest we use full command name e.g. `install` in our case.

File: routes.yaml (hunk 2 above)
Comment (Krzysztofoc) [reply routes.yaml]: 5th Feb conversation on the lfp-appeals slack channel

File: version (hunk 1 above)
Comment (Krzysztofoc) [reply version]: It's used by one of the pipeline steps and then it's passed to application versioning and GitHub tags. 
Every rebuild will add +1:
0.1.1
0.1.2 etc.

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+	npm run lint
+
+.PHONY: sonar
+sonar:
+#	npm run sonarqube
+
+.PHONY: test
+test: test-unit
+
+.PHONY: test-unit
```
Comment (n-r-b) [inline Makefile]: what's the reason for running sonarqube before the tests?

**Pull request merged**